moto/resourcegroupstaggingapi/models.py

```diff
@@ -18,6 +18,7 @@
 from moto.rds.models import RDSBackend, rds_backends
 from moto.redshift.models import RedshiftBackend, redshift_backends
 from moto.s3.models import S3Backend, s3_backends
+from moto.sqs.models import sqs_backends, SQSBackend
```

Comment:
```suggestion
from moto.sqs.models import SQSBackend, sqs_backends
```
I think this is the reason the linter fails: https://github.com/getmoto/moto/actions/runs/7092610448/job/19305827662?pr=7093

Reply:
I'll actually just merge it - let's see if that's the only thing that's blocking the tests